config/docker/postgresql/pg_hba.conf.sample

```diff
@@ -0,0 +1,89 @@
+# PostgreSQL Client Authentication Configuration File
```

Comment:
Hrm. Maybe we should chat on this next week. Feels like we're going a little overboard with the .sample/.override

If we want to keep things easily compatible with both docker machine and docker for mac, maybe requiring a password for the `safecast` user would be a reasonable compromise?

Reply:
This file and `config/docker/postgresql/postgresql.conf.sample` can be deleted. If you need them, you can copy from container instance.

About password, I am assuming this container is for local development, so I think it is not necessary to set password.

Reply:
Cool. Then let's have it default to `127.0.0.1:5432:5432` and provide a docker-compose.override.yml.sample that only specifies `5432:5432` for folks using docker-machine. We can add more examples if/when we have concrete use cases for other folks.

I'd also like to update https://github.com/safecast/safecastapi to match, then we can have docker be the recommended way for contributors to setup postgis.